.github/workflows/pull-request.yml

```
@@ -0,0 +1,265 @@
+
+      - name: resolve dependency
+        run: npm install -g @graphql-inspector/cli
+      
+      - name: Clone API Repository
+        run: |
+          # Retrieve the complete branch name directly from the GitHub context
+          FULL_BRANCH_NAME=${{ github.base_ref }}
+          echo "FULL_Branch_NAME: $FULL_BRANCH_NAME"
+          
+          # Clone the specified repository using the extracted branch name
+          git clone --branch $FULL_BRANCH_NAME https://github.com/PalisadoesFoundation/talawa-api && ls -a 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)